proxy/handler_one2many_test.go

```diff
@@ -628,8 +621,9 @@ func (s *ProxyOne2ManySuite) SetupSuite() {
 		return proxy.One2Many, result, nil
 	}
 
+	encoding.RegisterCodec(proxy.Codec())
```

Comment:
Grpc docs [recommends](https://github.com/grpc/grpc-go/blob/master/Documentation/encoding.md) to register codecs in the package which defines the codec.

Reply:
Since the name is `proto`, not sure if it is desirable, I would leave the control to the user, maybe they just mean to create a new `CodecWithParent` rather than registering. With `init`, it could cause trouble with import ordering regarding which codec gets to override the `proto` one

Another codec that does similar and does not use `init` either is `vtprotobuf`:

https://github.com/planetscale/vtprotobuf/blob/v0.6.0/codec/grpc/grpc_codec.go

But if you insist I will do it, let me know

Reply:
See my other comments.